src/modules/fundingManager/bondingCurve/tokens/ERC20Issuance_v1.sol

```diff
@@ -6,26 +6,28 @@ import {IERC20Issuance_v1} from
     "@fm/bondingCurve/interfaces/IERC20Issuance_v1.sol";
 
 // External Dependencies
-import {ERC20Upgradeable} from "@oz-up/token/ERC20/ERC20Upgradeable.sol";
-import {ContextUpgradeable} from "@oz-up/utils/ContextUpgradeable.sol";
-import {OwnableUpgradeable} from "@oz-up/access/OwnableUpgradeable.sol";
+import {ERC20} from "@oz/token/ERC20/ERC20.sol";
+import {Context} from "@oz/utils/Context.sol";
+import {Ownable} from "@oz/access/Ownable.sol";
```

Comment:
We usually use `Ownable2Step`, but I like the argument of being able to "throw the keys away" by setting it to `address(0)` - especially for the token. So we should probably keep it like this, even if we usually would use the 2-step variant. Does someone have a different viewpoint?

Reply:
So that was indeed one of the points, although tbh I didn't know that wasn't possible with `Ownabel2Step`. So from my point we can leave it like this

Reply:
Yeah I agree.